src/jemalloc/test/unit/mallctl.c

```diff
@@ -1,5 +1,11 @@
 #include "test/jemalloc_test.h"
 
+/*
+ * The minimal size of a pool. This must be large enough so that on systems with
+ * large amount of CPUs arena statistics can be allocated (32kb per arena).
+ */
+#define MALLCTL_MIN_SIZE (1 << 27) /* 128 megabytes */
```

Comment:
I would rather limit the number of arenas to 64 - same way as it was done in test/vmem_stats.
`export JE_VMEM_MALLOC_CONF="narenas:64"`
Could also be done via mallctl() call. Perhaps, it's even more appropriate for this test.